src/background/background-analytics.ts

```diff
@@ -0,0 +1,19 @@
+// Segment/Mixpanel libraries are not compatible with extension background
```

Comment:
_nit_ but I would go for doc style syntax for this larger comment

```
/*
Segment/Mixpanel libraries are not compatible with extension background
 
*/
```

Reply:
Unless we want the features of TSDoc, I'm a hardened line comment fan, following the age old advice from Douglas Cockford's The Best Parts

> What does Douglas Crockford say on the different types of comments in Javascript

> Douglas Crockford, a prominent figure in the JavaScript community and author of "JavaScript: The Good Parts," has strong opinions on comments in JavaScript. Here's a summary of his views on different types of comments:
> Single-line comments (//): Crockford generally approves of single-line comments. He considers them useful for brief explanations and clarifications.
> Multi-line comments (/* /): Crockford recommends against using multi-line comments. His main concern is that they can cause problems if used incorrectly, especially when commenting out code. If a developer accidentally includes a / within the commented block, it can lead to unexpected behavior.

Reply:
OK, good to know 👍 Good old Crockford